cmd/create/ocmrole/cmd.go

```diff
@@ -433,6 +437,9 @@ func buildCommands(prefix string, roleName string, rolePath string, permissionsB
 
 	linkRole := fmt.Sprintf("rosa link ocm-role --role-arn %s",
 		aws.GetRoleARN(accountID, roleName, rolePath))
+	if autoConfirmLink.String() == "true" {
+		linkRole += " -y"
+	}
```

Comment:
```suggestion
	if autoConfirmLink {
		linkRole += " -y"
	}
```
You can clean this up if you take in a bool instead of a flag